src/net/chat.rs

```diff
@@ -139,6 +140,7 @@ pub struct NeighborStats {
     pub block_push_rx_counts: VecDeque<(u64, u64)>, // (timestamp, num bytes)
     pub microblocks_push_rx_counts: VecDeque<(u64, u64)>, // (timestamp, num bytes)
     pub transaction_push_rx_counts: VecDeque<(u64, u64)>, // (timestamp, num bytes)
+    pub stackerdb_push_rx_counts: VecDeque<(u64, u64)>, // (timestamp, num bytes)
```

Comment:
This comment should be a rustdoc as well

Reply:
Fixed